OracleWebLogic/samples/12212-msiserver/container-scripts/launcher.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/bash
```

Comment:
Hi Avi,

launcher.sh invokes startManagedWebLogic.sh. This script (startManagedWebLogic.sh) like startWebLogic.sh used by other docker samples is created during WebLogic domain creation. I think these scripts communicate these signals to the underlying JVM (or not at the very least not inhibit the transmission of signal to the underlying JVM). WLS running in JVM handles SIGTERM and SIGKILL correctly.